certified-connectors/Maximizer/apiDefinition.swagger.json

```diff
@@ -0,0 +1,5059 @@
+            "name": "type",
+            "in": "query",
+            "type": "string",
+            "default": "Individual",
+            "description": "Type of AB Entry",
+            "enum": ["Company", "Individual", "Contact"],
+            "required": true,
+            "x-ms-visibility": "important",
+            "x-ms-summary": "Apply Action To"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "x-ms-dynamic-properties": {
+                "operationId": "ABAddrTxtbox",
+                "parameters": {
+                  "type": {
+                    "parameterReference": "type"
+                  }
+                },
+                "itemValuePath": "Schema"
+              },
+              "x-ms-dynamic-schema": {
+                "operationId": "ABAddrTxtbox",
+                "parameters": {
+                  "type": {
+                    "parameter": "type"
+                  }
+                },
+                "value-path": "Schema"
+              },
+              "properties": {}
+            }
+          }
+        ],
+        "description": "Updates an existing AB Entry Address record",
+        "x-ms-visibility": "important"
+      }
+    },
+    "/api/DropdownCity": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK, Success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Code": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Code"
+                },
+                "State": {
+                  "type": "string",
+                  "description": "State"
+                },
+                "Values": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "sample": {
+                        "type": "string",
+                        "description": "sample"
+                      },
+                      "value": {
+                        "type": "string",
+                        "description": "value"
+                      },
+                      "label": {
+                        "type": "string",
+                        "description": "label"
+                      }
+                    }
+                  },
+                  "description": "Values"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Dropdown list for City",
+        "description": "DropdownCity",
+        "operationId": "DropdownCity",
+        "x-ms-visibility": "internal",
+        "parameters": []
+      }
+    },
+    "/api/DropdownStateProvince": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}

```

Comment:
@MaximizerIntegration empty schema in not allowed. please define a proper schema.